web/src/components/apps/AppDetailPage.tsx

```diff
@@ -380,6 +380,18 @@ function AppDetailPage(props: Props) {
     }
   }
 
+  const resetRedeployErrorMessage = () => {
```

Comment:
this is not great but opting to get the quick fix in for now until we refactor the API calls.